packages/expect/src/__tests__/__snapshots__/toThrowMatchers.test.js.snap

```diff
@@ -383,17 +393,27 @@ Received message: <red>\\"apple\\"</>
 exports[`toThrowError error class threw, but class did not match (non-error falsey) 1`] = `
 "<dim>expect(</><red>received</><dim>).</>toThrowError<dim>(</><green>expected</><dim>)</>
 
-Expected name: <green>\\"Err2\\"</>
+Expected constructor: <green>Err2</>
 
 Received value: <red>undefined</>
 "
 `;
 
+exports[`toThrowError error class threw, but class should not match (error subclass) 1`] = `
+"<dim>expect(</><red>received</><dim>).</>not<dim>.</>toThrowError<dim>(</><green>expected</><dim>)</>
+
+Expected constructor: not <green>Err</>
+Received constructor:     <red>SubErr</>
+
+Received message: <red>\\"apple\\"</>
+
+      <dim>at jestExpect (</>packages/expect/src/__tests__/toThrowMatchers-test.js<dim>:24:74)</>"
```

Comment:
Is this supposed to be here?

Reply:
Good eyes! There are 32 occurrences in the `.snap` file from `CustomError` base class near beginning of `toThrowMatchers.test.ts` file:

```js
// Custom Error class because node versions have different stack trace strings.
class customError extends Error {
  constructor(message) {
    super();
    this.message = message;
    this.name = 'Error';
    this.stack =
      'Error\n' +
      '  at jestExpect' +
      ' (packages/expect/src/__tests__/toThrowMatchers-test.js:24:74)';
  }
}
```